app/ai_backend/test_genesis_connector.py

```
@@ -990,15 +1922,20 @@ def __str__(self):
             'message': 'test'
         }
         
-        formatted = self.connector.format_payload(payload)
+        for thread in threads:
```

Ruff (0.11.9)

Undefined name `threads`

(F821)